strr-examiner-web/nuxt.config.ts

```diff
@@ -0,0 +1,90 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+  devtools: { enabled: false },
+  ssr: false,
+
+  future: {
+    compatibilityVersion: 4
+  },
+
+  modules: [
+    '@nuxtjs/eslint-module',
+    '@nuxt/test-utils/module',
+    '@nuxt/image'
+  ],
+
+  i18n: {
+    locales: [
+      {
+        name: 'English',
+        code: 'en-CA',
+        iso: 'en-CA',
+        dir: 'ltr',
+        file: 'en-CA.ts'
+      },
+      {
+        name: 'Français',
+        code: 'fr-CA',
+        iso: 'fr-CA',
+        dir: 'ltr',
+        file: 'fr-CA.ts'
+      }
+    ],
+    strategy: 'prefix',
+    lazy: true,
+    langDir: 'locales',
+    defaultLocale: 'en-CA',
+    detectBrowserLanguage: false,
+    vueI18n: './i18n.config.ts'
+  },
+
+  extends: [
```

Comment:
Keep in mind extends will bring over everything (pages/components/utils/composables/etc), so all the pages that are in the strata/platform/host app will be available in the examiner view this way. 

You can ignore certain paths from being prerendered but in this case we should only need to extend the base layer. Mostly everything thats reusable has been added to the base layer. If there are specific things you'd like from the other apps we can look into moving those to the base layer.